aioshelly/rpc_device/device.py

```diff
@@ -26,6 +26,7 @@
     DEVICE_POLL_TIMEOUT,
     FIRMWARE_PATTERN,
     MODEL_BLU_GATEWAY_GEN3,
+    MODEL_X_BASE,
```

Comment:
```suggestion
```

Reply:
I think we can keep it to use on HA side to filter Bluetooth feature for this class of devices

Reply:
I think that we are assuming that all X models will have this part number, but this is not correct, we are also missing a `ShellyDevice` for this added model (not for the `jwt` model, for the base model)